src/components/OutsideNavbar.tsx

```
@@ -13,12 +13,15 @@ import {
   ListItemText,
   Tabs,
   Divider,
-  Typography
+  Typography,
```

ESLint

[error] 'Typography' is defined but never used.

(@typescript-eslint/no-unused-vars)